bool use_file
float64 alpha
float64 t_init_double_support
float64 t_double_support
float64 t_step
float64 t_final_double_support
float64 step_length
float64 step_width
float64 COM_height
int8 step_count
float64 ankle_height
float64 dt
float64 theta
float64 step_height
---
int32 traj_size
bool result
